app/templates/receive_package.hbs

```diff
@@ -1,64 +1,70 @@
 <div id="{{concat 'stockitAddItemModal' package.id}}" class="stockitAddItemModal ui">
 
-  <section class="main-section receive_package_modal">
+  <nav class="tab-bar new-item-nav">
+    <section class="tab-bar-section">
+      <h1 class="title">Receive-{{package.packageName}}</h1>
```

Comment:
`Receive` can be moved to translations and you can pass value of package.packageName also